### PR TITLE
Fixing parquet benchmarks

### PR DESCRIPTION
**Pull request by rgsl888prabhu**
`return_filemetadata` was removed in one of the recent PR, and missed to remove it in benchmarks.

### CODE REVIEW

File: cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp [hunk 1]
```diff
@@ -63,9 +63,9 @@ void BM_parq_write_varying_inout(benchmark::State& state)
 
 void BM_parq_write_varying_options(benchmark::State& state)
 {
-  auto const compression     = static_cast<cudf::io::compression_type>(state.range(0));
-  auto const enable_stats    = static_cast<cudf::io::statistics_freq>(state.range(1));
-  auto const output_metadata = state.range(2) != 0;
+  auto const compression  = static_cast<cudf::io::compression_type>(state.range(0));
+  auto const enable_stats = static_cast<cudf::io::statistics_freq>(state.range(1));
+  auto const file_path    = state.range(2) != 0 ? "dummy_path.parquet" : "";
```
Comment (codereport) [inline cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp:68]: Dummy is on the ["non-inclusive" language list](https://developers.google.com/style/inclusive-documentation). 
```suggestion
  auto const file_path    = state.range(2) != 0 ? "unused_path.parquet" : "";
```

**Review by codereport — CHANGES_REQUESTED** [review]

**Review by codereport — APPROVED** [review]
lgtm :+1:

**Review by cwharris — APPROVED** [review]

**Comment by harrism** [comment]
@gpuciboth merge

**Comment by harrism** [comment]
@gpucibot merge

**Pull request merged**